server/export/exportDetail.js

```diff
@@ -0,0 +1,71 @@
+const Excel = require('exceljs');
+const path = require('path');
+const moment = require('moment');
+const _ = require('lodash');
+const { getUser } = require('../service/userService');
+const {
+  findUserTripsForRange,
+  isDayOfMonth,
+  enumarateTripsWorkingDays,
+  enumarateWorkingDays
+} = require('../service/tripService');
+const {
+  findUserLeavesForRange,
+  enumarateLeavesWorkingDays
+} = require('../service/leaveService');
+
+module.exports = async ({ userId, month }) => {
+  console.log('export to detail', userId, month);
+
+  const user = await getUser(userId);
+  const m_month = moment(month, 'YYYY-MM');
+  const dateRange = [
+    m_month.toDate(),
+    m_month
+      .clone()
+      .add(1, 'month')
+      .toDate()
+  ];
+
+  const isThisMonth = d => isDayOfMonth(d, month);
```

Comment:
The usage of this was not obvious without reading the function and going throught all the possible values in the head, comment would be helpful.